src/victory-util/add-events.js

```diff
@@ -0,0 +1,61 @@
+import { defaults, assign, isFunction, partialRight } from "lodash";
+import Events from "./events";
+
+export default (WrappedComponent) => {
+  return class addEvents extends WrappedComponent {
+    constructor() {
+      super();
+      this.state = {};
+      const getScopedEvents = Events.getScopedEvents.bind(this);
+      this.getEvents = partialRight(Events.getEvents.bind(this), getScopedEvents);
+      this.getEventState = Events.getEventState.bind(this);
+    }
+
+    componentWillMount() {
```

Comment:
so this would potentially break components that defined their own `componentWillMount`, `componentWillReceiveProps` or use `state` (because it's overwritten in the constrcutor). are you so sure that will never be used in components? would it be worth it to at least `console.error` if eg. `WrappedComponent.prototype.componentWillMount` is a function?